src/app/app-design-tool/design-tool.component.scss

```diff
@@ -0,0 +1,36 @@
+// Copyright 2018 Novo Nordisk Foundation Center for Biosustainability, DTU.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+mat-sidenav-container {
+  height: 100%;
+}
+
+mat-sidenav {
+  width: 280px;
+  mat-checkbox {
+    display: inherit !important;
+  }
+}
+
+.container {
+  height: 100%;
+}
+
+.floating-menu {
```

Comment:
If this is the same as in interactive-map, you could turn this into either a separate component in the global namespace (right in `app/components` or add this style as a global style